### DISCUSSION TITLE
链接的srpc.Lib是哪个库的

### DISCUSSION THREAD

**Discussion by samueltian666** [Wei1030/2Box#16] [Q&A] [answered]
.............

**Comment by wanwanzaizai** [comment]
同求

**Comment by marklove5102** [comment]
sRPC.lib  vs2019不能用. 估计是vs2015?无环境无法测试了.
作者22年无github 活跃...........

**Comment by Wei1030** [comment, marked as answer]
https://github.com/Wei1030/common_binary/releases/download/v1.0/sRPC-static-lib.zip
只能用vs2008
